distribution/test/itests/pom.xml

```diff
@@ -78,7 +80,7 @@
             <dependency>
                 <groupId>junit</groupId>
                 <artifactId>junit</artifactId>
-                <version>4.11</version>
+                <version>${junit.version}</version>
```

Comment:
No idea, I only heard about it from others. I think @ahoffer tried it though, he may know. It'd be great if this was just a result from the changes in the pr though!